<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Profile :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DirectorySelectWidget" name="profile_widget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Custom preferences:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PreferencesWidgetEditor" name="pref_widget" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Custom addons:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="AddonsWidgetEditor" name="addons_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectorySelectWidget</class>
   <extends>QWidget</extends>
   <header>mozregui.utils</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreferencesWidgetEditor</class>
   <extends>QWidget</extends>
   <header>mozregui.pref_editor</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddonsWidgetEditor</class>
   <extends>QWidget</extends>
   <header>mozregui.addons_editor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
